### DISCUSSION TITLE
Disable build email?

### DISCUSSION THREAD

**Discussion by leanderlee** [vercel/community#36] [Help] [answered]
Hi, I'm sure to have just overlooked something, but I couldn't find an answer to this. Is there a way to disable the now[bot] build emails? They're getting spammy and clicking the unsubscribe in the email only disables the updates for that commit.

**Comment by wiiiimm** [comment]
These emails are generated by GitHub as our bot leave comments to your commits and PRs wgeb it build and deploy your code. You can look at https://github.com/settings/notifications to control what kind of notifications make it to your email inbox.

You can also look into the [github.silent](https://vercel.com/docs/concepts/projects/project-configuration/git-configuration#github.silent) configuration to prevent our bot from commenting on your commits and PRs.

**Reply by WesleyYue** [reply]
1. I don't see anything in the current docs about github.silent
2. Asking users to turn off all Github notifications on conversations is not a reasonable solution. The problem is Vercel bot spamming every conversation. I want conversation notifications from Github, just not when it's Vercel bot.

**Reply by amyegan** [reply]
`github.silent` documentation can be found at https://vercel.com/docs/concepts/projects/project-configuration/git-configuration#github.silent

**Comment by leanderlee** [comment]
Ah, that was it. Thanks!

**Comment by GorvGoyl** [comment]
@williamli I created `vercel.json` file and added below config but still that bot is commenting and email notification is getting triggered.
```
{
  "github": {
    "silent": true
  }
}
```

**Comment by shamwela** [comment]
How can I turn off all notifications without having to create `vercel.json` for every new project?

**Comment by oluckyman** [comment]
These should be disabled by default. There is enough noise in the world already.

**Reply by wiiiimm** [reply]
You can reach out to GitHub and ask them to consider switching their default for notifications.

**Comment by MathiasReker** [comment]
You can disable notifications in settings: https://github.com/settings/notifications

**Comment by wiiiimm** [comment, marked as answer]
@oluckyman @shamwela @GorvGoyl Please check the sender of these build emails. If they are coming from GitHub then you should look at your GitHub notification settings at https://github.com/settings/notifications. You can control how different GitHub notifications are delivered.

![image](https://user-images.githubusercontent.com/179761/147821145-257954a7-85cc-49b3-b06e-d89c42d86956.png)

**Comment by bdlowery** [comment]
How can I disable the emails from vercel bot commenting on a commit I make. There's nothing in the notification settings (that i can see) that says anything about comments on commits. I've already disabled github action emails.

The comment marked as an answer isn't really an answer. It's just a picture of the github notification settings page.

**Reply by amyegan** [reply]
Are the emails you see coming from a Vercel address, or from a GitHub address?

**Reply by amyegan** [reply]
Like William mentioned in an earlier comment, you can use [github.silent](https://vercel.com/docs/concepts/projects/project-configuration/git-configuration#github.silent) in your [Project Config](https://vercel.com/docs/project-configuration) to prevent the automatic comment on your PR. But if you still want to get the comments for build status on each PR, then you will need to go to your GitHub Notification settings at https://github.com/settings/notifications and uncheck "Comments on Issues and Pull Requests" under "Email notification preferences". This would stop email notifications for all comments on issues and pull requests.

**Comment by ayunami2000** [comment]
Hi, other users are spamming my email with vercel successful deployment emails. There is literally no way to turn them off, because we do not use vercel at all; they are triggered by other users. I do not want to change the global GitHub settings just because vercel[bot] is spamming my email. Please, at least allow me to provide my repo and have you guys blacklist the repo. We do not use vercel, so it would be quite silly to include vercel.json in a project that again does not use vercel at all.

**Reply by ayunami2000** [reply]
Quote from a friend who has the same issue: "every day someone new makes a vercel of (project) and they literally just pick a random folder"
